### DISCUSSION TITLE
Thanks & a suggestion

### DISCUSSION THREAD

**Discussion by chriscjcj** [CarlGao4/Demucs-Gui#12] [General]
I wanted to add a comment to [the issue I posted](https://github.com/CarlGao4/Demucs-Gui/issues/11), but doing so would have closed the issue so I am posting here.

I feel rather silly. It never occurred to me to just click on the "Load model" button. When the app finished launching, I inferred that the model that was listed in the model path was already loaded by default. When I didn't see any other way to make the app do anything, I just erroneously surmised it was due to the missing components that were reported. And as you elucidated, the lack of either ffmpeg or cuda is actually moot. I just figured that the only reason to click the "Load model" button would be to load a model other than the default.

Thank you for telling this very obtuse user that I needed to hit that button.

So, yes... I'm not the sharpest tool in the shed. :-)  I would, however, submit that perhaps a tweak could be made to that window to indicate to the user that he/she must hit that "Load model" button in order to proceed to the app's raison d'être. Or, perhaps you could load a default model at startup like I was expecting it to; not sure if that would involve unintended consequences.

In any case, I was able to get everything to work and I'm thrilled with the results so far. Back in the 80's I used to create mashups before they were called mashups. I would find two songs with the same BPM and key, play the instrumental of one and the acappella of the other. Of course, only a small percentage of 12" singles had instrumentals and acappellas. With this tool, I can create both. And now that I can manipulate pitch and speed independently of each other, the possibilities are endless. You have my most sincere gratitude for all the work you've done to make GUI for this on the Mac. Thank you so very much.

**Comment by CarlGao4** [comment]
Thanks for your support. 

As you can see, loading a model requires a lot of time and memory, so loading a model as default would make the start up process much slower. Besides, I haven't found a way to free all the memory of a model, there is always about 800MB of memory left, so it would be wasteful if the user want to use another model. 

I will add a manual page in the next version to tell the users to click `load model`.

**Comment by CarlGao4** [comment]
I’m preparing the document but have no idea at this point. Feel free to come up with any suggestions at #23!
